CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.21.0
+- Allow use with future unknown Rails versions
```

Comment:
Related to my comment about the upper bound on the version constraint in the gemspec, I don't think we want this.